cmd/pd-server/main.go

```diff
@@ -27,17 +28,24 @@ import (
 )
 
 var (
-	config                 = flag.String("c", "", "config file")
-	addr                   = flag.String("addr", "127.0.0.1:1234", "server listening address")
-	advertiseAddr          = flag.String("advertise-addr", "", "server advertise listening address [127.0.0.1:1234] for client communication")
-	httpAddr               = flag.String("http-addr", ":9090", "http server listening address")
-	rootPath               = flag.String("root", "/pd", "pd root path in etcd")
-	leaderLease            = flag.Int64("lease", 3, "leader lease time (second)")
-	logLevel               = flag.String("L", "debug", "log level: info, debug, warn, error, fatal")
-	tsoSaveInterval        = flag.Int64("tso-save-interval", 2000, "the interval time (ms) to save timestamp")
-	clusterID              = flag.Uint64("cluster-id", 0, "initial cluster ID for the pd cluster")
-	maxPeerCount           = flag.Uint64("max-peer-count", 3, "max peer count for the region")
-	metricAddr             = flag.String("metric-addr", "", "metric address")
+	config = flag.String("c", "", "config file")
+
+	clusterID           = flag.Uint64("cluster-id", 0, "initial cluster ID for the pd cluster")
+	name                = flag.String("name", "pd", "human-readable name for this pd member")
+	dataDir             = flag.String("data-dir", "", "path to the data directory (default 'default.{name}')")
+	host                = flag.String("host", "127.0.0.1", "host for outer traffic")
+	clientPort          = flag.Uint64("client-port", 2379, "port for client traffic")
+	advertiseClientPort = flag.Uint64("advertise-client-port", 0, "advertise port for client traffic (default 2379)")
```

Comment:
yes, using 0 here is just easy for laster adjustUint64 function. 
if we set 2379 here, and use another port in clientPort and forget to change advertise port, the advertise port will be 2379, it is wrong.